src/Evolution/EventsAndDenseTriggers/DenseTriggers/Filter.hpp

```diff
@@ -62,29 +63,31 @@ class Filter : public DenseTrigger {
   explicit Filter(std::unique_ptr<DenseTrigger> trigger,
                   std::unique_ptr<Trigger> filter);
 
-  using is_triggered_argument_tags = tmpl::list<Tags::DataBox>;
+  using is_triggered_return_tags = tmpl::list<Tags::DataBox>;
+  using is_triggered_argument_tags = tmpl::list<>;
```

Comment:
If `is_triggered` is being called with a `db::mutate_apply` now, and you pass in the box as a return tag, are you allowed to do `db::mutate` inside `is_triggered`? I thought no?

Reply:
It's a special case added in #5478.  The `db::mutate` and `db::mutate_apply` functions don't lock the box if they don't actually mutate anything.  So you can take the box as a return, but you can't take anything else at the same time.